config/services.go

```diff
@@ -75,6 +75,9 @@ func (a *AppConfig) LoadTestServices(ctx context.Context) (*AppServices, error)
 	// Start services
 	_services := new(AppServices)
 
+	nopLogger := zerolog.Nop()
+	_services.Logger = &nopLogger
```

Comment:
Why we're disabling logger in tests? it's harder to debug tests without logger (because you always need a debugger for that)

Reply:
We do it almost everywhere in tests in spv-wallet. So I've just adjusted to our "standard" instead of introduced something new.
![image](https://github.com/user-attachments/assets/5741afd1-79c4-4636-990e-db01aebe2264)

Reply:
ok, then. Let's leave it.